BusinessTripApplication/Content/bootstrap.css

```diff
@@ -1,81 +1,154 @@
 ﻿/*!
- * Bootstrap v3.3.7 (http://getbootstrap.com)
- * Copyright 2011-2016 Twitter, Inc.
+ * Bootswatch v4.1.2
```

Comment:
Bootstrap files should not be modified. If you want to make changes to bootstrap classes you should override them in separate files.